test/unit/tech/tech.test.js

```diff
@@ -1,5 +1,6 @@
 var noop = function() {}, clock, oldTextTracks;
 
+import videojs from '../../../src/js/video.js';
```

Comment:
over here since we don't need all of videojs, we can import just the `extendsFn` itself: https://github.com/videojs/video.js/blob/dd9930b6c68a1016475b35d1f05c4421f83649bb/src/js/video.js#L18